issue280and524.nimble

```diff
@@ -10,15 +10,10 @@ srcDir        = "src"
 
 requires "nim >= 0.18.0"
 
-import strutils
-
 task setup, "Download and generate":
   let
-    dirSep = when defined(windows): "\\" else: "/"
-    path =
-      if selfExe().len == 0: ""
-      else: selfExe().rsplit(dirSep, maxsplit = 1)[0] & dirSep
-  exec path & "nimble install https://github.com/nimble-test/issue280and524.git?subdir=generator -y"
+    nimble = getEnv("NIMBLE_TEST_BINARY_PATH", "nimble")
```

Comment:
i'd rather this fail then choose some random version of Nimble

Reply:
Done